bitsv/network/block.py

```diff
@@ -0,0 +1,29 @@
+
+
+
+class Block:
+    """Represents a block returned from the network."""
+
+    __slots__ = ('hash', 'height', 'prev', 'next', 'txs')
+
+    def __init__(self, hash, height, prev, next, txs):
+        self.hash = hash
+        self.height = height
+        self.prev = prev
+        self.next = next
+        self.txs = txs
+
```

Comment:
where's the merkle root? It's available in the json response from both whatsonchain and bsvbookguarda.